wasm/engine.go

```diff
@@ -38,6 +38,10 @@ func (engine *Engine) initGame(memlist []byte, bankFilesMap map[int][]byte) {
 }
 
 func (engine *Engine) mainLoop() {
+	if engine.keyMap[anotherworld.KeyPause] == true {
```

Comment:
Should pause be a "hold" or a state in the engine that we toggle? Press to pause press again to unpause? feels more common.

Reply:
Also if we had an engine state wouldn't that allow a more efficent pause state? the hal could check and slow down the blit while paused (no point rerendering the same frame over and over no?)

Reply:
My goal is to solve this with #45

Reply:
yeah.. used id because i know shit about css ;)

Reply:
fixed btw